Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -1544,14 +1548,15 @@ void EditorActionsHandler::OnToolBarRegistrationHook()
         toolBarProperties.m_name = "Play Controls";
         m_toolBarManagerInterface->RegisterToolBar(PlayControlsToolBarIdentifier, toolBarProperties);
     }
-
-    // Set the toolbars
-    m_mainWindow->addToolBar(Qt::ToolBarArea::TopToolBarArea, m_toolBarManagerInterface->GetToolBar(ToolsToolBarIdentifier));
-    m_mainWindow->addToolBar(Qt::ToolBarArea::TopToolBarArea, m_toolBarManagerInterface->GetToolBar(PlayControlsToolBarIdentifier));
 }
 
 void EditorActionsHandler::OnToolBarBindingHook()
 {
+    // Add ToolBars to ToolBar Areas
+    // We space the sortkeys by 100 to allow external systems to add toolbars in-between.
+    m_toolBarManagerInterface->AddToolBarToToolBarArea(EditorMainWindowTopToolBarAreaIdentifier, ToolsToolBarIdentifier, 100);
```

Comment:
suggestion: Might it be worth having a `ToolBarIdentifierSpacing` constant set to 100 and then we add that * `1/2/3/etc` to the starting one? Just might nice to have that rather than explicit values everywhere

Reply:
I'll consider this across all APIs for a follow-up